src/client/src/containers/AzureSubscriptions/index.tsx

```diff
@@ -110,29 +118,74 @@ class AzureSubscriptions extends React.Component<Props, IState> {
     }
     return () => {};
   }
+
+  public getServicesOrganizer(
+    type: string | undefined,
+    isLoggedIn: boolean,
+    setDetailPage: any,
+    title: any
+  ) {
+    const { formatMessage } = this.props.intl;
+    return (
+      <div
+        className={classnames(styles.servicesContainer, {
+          [styles.overlay]: !isLoggedIn
+        })}
+      >
+        <div className={styles.servicesCategory}>
+          <div className={styles.categoryTitle}>{type}</div>
+          <div className={styles.categoryDescriptor}>
+            {formatMessage(title)}
+          </div>
+          <div className={styles.servicesCategoryContainer}>
+            {azureServiceOptions.map(option => {
+              if (option.type === type) {
+                return (
+                  <div
+                    key={JSON.stringify(option.title)}
+                    className={classnames(styles.subscriptionCardContainer)}
+                  >
+                    <Card
+                      option={option}
+                      buttonText={this.addOrEditResourceText(
+                        option.internalName
+                      )}
+                      handleButtonClick={this.getServicesModalOpener(
+                        option.internalName
+                      )}
+                      disabled={!isLoggedIn}
+                      handleDetailsClick={setDetailPage}
+                      useNormalButtons={this.isSelectionCreated(
+                        option.internalName
+                      )}
+                    />
+                  </div>
+                );
+              }
+            })}
+          </div>
+        </div>
+      </div>
+    );
+  }
   public render() {
     const { isLoggedIn, setDetailPage } = this.props;
+    const serviceTypes: any[] = [];
+    azureServiceOptions.map(option => {
```

Comment:
think you can just do a .forEach here since you aren't mapping the elements to a new format.